SupClient/src/setupHotkeys.ts

```diff
@@ -1,3 +1,6 @@
+let currentHelpCallback: Function = null;

```

Comment:
I think the callback is unlikely to change so I'd simply call it `helpCallback`.